src/database/sql-files/sqlFilesIndex.js

```diff
@@ -0,0 +1,12 @@
+const QueryFile = require('pg-promise').QueryFile
```

Comment:
really minor point but can we do const {QueryFile} = require('pg-promise') just for consistency as we seem to have done that for most of the project?

Reply:
I agree